extensions/amp-twitter/1.0/storybook/Basic.js

```diff
@@ -67,14 +67,12 @@ export const timelines = () => {
   const timelineUserId = '3450662892';
   return (
     <Twitter
-      bootstrap="https://3p.ampproject.net/2104170104001/vendor/twitter.js"
```

Comment:
Why is this part of the PR?

Reply:
This was being passed in because `INTERNAL_RUNTIME_VERSION` wasn't previously available.

Reply:
Actually... I'm going to remove this separately, since there's a few like this sprinkled around and they may involve other changes.